### PR TITLE
💪 Add RN 0.60.x fitness + update readme

### PR DESCRIPTION
**Pull request by flaming-codes**
- added podspec to make lib for autolinking in RN >= 0.60.0 usable
- updated readme to reflect installation changes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,5 +47,9 @@
   },
   "jest": {
     "preset": "react-native"
-  }
+  },
+  "files": [
+    "ios",
+    "react-native-home-indicator.podspec"
+  ]
```
Comment (michaelknoch) [inline package.json:54]: i think we should also include the main `index.js` to this list

Comment (flaming-codes) [reply package.json:54]: 'main' is included by default: [link](https://docs.npmjs.com/files/package.json#files)

I've tested my fork with the changes in a fresh 0.60.4 repo, and the npm install + pod install work as expected.

Comment (michaelknoch) [reply package.json:54]: ok, good to know

**Comment by michaelknoch** [comment]
thx @CaptainQuark

**Review by michaelknoch — APPROVED** [review]

**Pull request merged**